src/toM3u8.js

```diff
@@ -151,8 +200,10 @@ export const toM3u8 = dashPlaylists => {
   const vttOnly = ({ attributes }) =>
     attributes.mimeType === 'text/vtt' || attributes.contentType === 'text';
 
-  const videoPlaylists = dashPlaylists.filter(videoOnly).map(formatVideoPlaylist);
-  const audioPlaylists = dashPlaylists.filter(audioOnly);
+  const videoPlaylists = mergeDiscontiguousPlaylists(
+    dashPlaylists.filter(videoOnly)
+  ).map(formatVideoPlaylist);
+  const audioPlaylists = mergeDiscontiguousPlaylists(dashPlaylists.filter(audioOnly));
```

Comment:
`formatVideoPlaylist` and `formatAudioPlaylist` remove periodIndex; do we want to leave it in the final M3U8 playlists?

Reply:
That's expected, I don't want to expose more than is necessary